azure-functions/acbs-function/activity-get-facility-master/index.js

```diff
@@ -12,28 +12,33 @@ const api = require('../api');
 const { isHttpErrorStatus } = require('../helpers/http');
 
 const getFacilityMaster = async (context) => {
-  const { facilityId } = context.bindingData;
+  try {
+    const { facilityId } = context.bindingData;
 
-  if (facilityId) {
-    const { status, data, headers: { etag } } = await api.getFacility(facilityId);
+    if (facilityId) {
+      const { status, data, headers: { etag } } = await api.getFacility(facilityId);
 
-    if (isHttpErrorStatus(status)) {
-      throw new Error(
-        JSON.stringify({
-          name: 'ACBS Facility fetch error',
-          facilityId,
-          dataReceived: data,
-        }, null, 4),
-      );
+      if (isHttpErrorStatus(status)) {
+        throw new Error(
+          JSON.stringify({
+            name: 'ACBS Facility fetch error',
+            facilityId,
+            dataReceived: data,
+          }, null, 4),
+        );
+      }
+
+      return {
```

Comment:
is there a check to make sure that data exists?

Reply:
If null or empty data is returned then the further checks on the calling function will not be further executed, so no check is necessary here.